src/apis/admin.ts

```diff
@@ -0,0 +1,509 @@
+    description?: string,
+    created_at?: Date,
+    last_updated_at?: Date,
+    defaults?: Record<string, any> | null,
+    users?: Record<string, any>[],
+}
+
+export interface WorkspacesListParams {
+    page_size?: number;
+    current_page?: number;
+}
+
+export interface WorkspacesListResponse extends APIResponseType {
+    total?: number,
+    object?: string,
+    data?: WorkspacesGetResponse[]
+}
+
+export interface WorkspacesUpdateParams{
+    workspaceId?: string,
+    name?: string,
+    description?: string,
+    defaults?: Record<string, any>,
+}
+
+export interface WorkspacesUpdateResponse extends APIResponseType {
+    id?: string,
+    slug?: string,
+    name?: string,
+    description?: string,
+    created_at?: Date,
+    is_default?:number,
+    last_updated_at?: Date,
+    defaults?: Record<string, any>,
+    object?: string,
+}
+
+export interface WorkspacesDeleteParams{
+    workspaceId?: string;
+    name?: string;
+}
+
+export interface WorkspaceMemberAddParams{
+    workspaceId?: string,
+    users?: { id: string, role: "member" | "admin" }[],
+}
+
+export interface WorkspaceMemberGetParams{
+    workspaceId?: string,
+    userId?: string,
+}
+
+export interface WorkspaceMemberGetResponse extends APIResponseType {
+    first_name?: string,
+    last_name?: string,
+    org_role?: string,
+    role?: string,
+    created_at?: Date,
+    last_updated_at?: Date,
+    status?: string
+    workspace_id?: string,
+    scopes?: string[],
+    settings?: Record<string, any>| null,
+    object?: string,
+}
+
+export interface WorkspaceMemberListParams {
+    workspaceId?: string,
+    page_size?: number,
+    current_page?: number,
+    email?: string,
+    role?: "admin" | "manager" | "member" | any,
+}
+
+export interface WorkspaceMemberListResponse extends APIResponseType {
+    total?: number,
+    object?: string,
+    data?: WorkspaceMemberGetResponse[]
+}
+
+export interface WorkspaceMemberDeleteParams{
+    workspaceId?: string,
+    userId?: string,
+}
+
+export interface WorkspaceMemberUpdateParams{
+    workspaceId?: string,
+    userId?: string,
+    role?: "admin" | "member" | any,
+}
+// Function to convert UsersGetParams to query parameters
+function toQueryParams(params?: (UsersListParams | UserInviteListParams | WorkspacesListParams | WorkspaceMemberListParams)): string {
+    if (!params) {
+        return '';
+    }
+    const queryParams = Object.entries(params)
+        .filter(([, value]) => value !== undefined && value !== null)
+        .map(([key, value]) => `${key}=${value}`)
+        .join('&');
+    
+    return queryParams ? `?${queryParams}` : '';
+}
+
+export class Admin extends ApiResource {
+    users: Users
+    workspaces: Workspaces
+    constructor(client: any) {
+        super(client);
+        this.users = new Users(client);
+        this.workspaces= new Workspaces(client);
+    }
+}
+
+
+export class Users extends ApiResource {
+
+    invites: Invites
+
+    constructor(client: any) {
```

Comment:
Can remove the constructor and implement it in this way 

`invites: Invites = new Invites(this.client);`